crypto/bls/public_test.go

```diff
@@ -36,3 +36,22 @@ func TestPublicKeyBytes(t *testing.T) {
 	require.Equal(pk, pk2)
 	require.Equal(pkBytes, pk2Bytes)
 }
+
+func TestAggregatePublicKeysNoop(t *testing.T) {
```

Comment:
Can we add a test that aggregates 2 public keys and 2 signatures and verifies the aggregate signature with the aggregate public key?